server/api/controllers/userController.js

```diff
@@ -6,16 +7,22 @@ import User from '../models/user';
 import createToken from '../helpers/authToken';
 import HashedPassword from '../helpers/hashPassword';
 
-class Users{
+class Users {
 	static async signup(req, res) {
 		// eslint-disable-next-line camelcase
 		const {
-			email, first_name, last_name, password,
+			email,
+			first_name,
+			last_name,
+			password,
 		} = req.body;
 
 		const passwordHashed = HashedPassword.hashPassword(password);
 		const newUser = User.createNewUser({
-			email, first_name, last_name, passwordHashed,
+			email,
+			first_name,
+			last_name,
+			passwordHashed,
 		});
 		// eslint-disable-next-line max-len
```

Comment:
It isn't a best practice to every time disable eslint's rules in your code

Reply:
I have fixed most of them now